_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/upload_images.yml

```diff
@@ -13,8 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+  - name: Set ciao-cli location
+    set_fact: ciao_cli={{ (gopath + '/bin/ciao-cli') if ciao_dev else '{{ playbook_dir }}/fetch/ciao-cli' }}
```

Comment:
```playbook_dir``` contains the playbook base directory. http://docs.ansible.com/ansible/playbooks_variables.html